README.md

```diff
@@ -586,6 +588,29 @@ createServer(middleware).listen(3000);
         Custom path to match requests against. Defaults to <code>/api/github/webhooks</code>.
       </td>
     </tr>
+    <tr>
+      <td>
+        <code>pathRegex</code>
```

Comment:
What do you think of accepting a `path` and allowing it to be either a `string` or a `RegExp`? That interface feels nicer to me - although I'm not an expert in idiomatic JavaScript. It avoids the "override" complexity you have here, which could lead to surprises.

Reply:
I'm not a js pro either, but something like a `string | RegExp` and later on a type matching to them seems to be more dangerous to me. But I can try it, a typeof string should split them correctly.

Reply:
Let’s wait and see what @gr2m or other maintainers think. 

If we keep it this way, we should enforce that exactly one of the path options is provided.

Reply:
> I'm not a js pro either, but something like a `string | RegExp` and later on a type matching to them seems to be more dangerous to me.

Can you explain why you think so?

I don't see the need to add another argument that has the exact same purpose

Reply:
I fixed this also.